Marlin/language_pl.h

```diff
@@ -32,136 +32,136 @@
+#define MSG_NO_CARD                         _UxGT("Brak karty")
+#define MSG_DWELL                           _UxGT("Uspij...")
+#define MSG_USERWAIT                        _UxGT("Oczekiwanie...")
+#define MSG_RESUMING                        _UxGT("Wznawianie druku")
+#define MSG_PRINT_ABORTED                   _UxGT("Druk przerwany")
+#define MSG_NO_MOVE                         _UxGT("Brak ruchu")
+#define MSG_KILLED                          _UxGT("Ubity. ")
+#define MSG_STOPPED                         _UxGT("Zatrzymany. ")
+#define MSG_CONTROL_RETRACT                 _UxGT("Wycofaj mm")
+#define MSG_CONTROL_RETRACT_SWAP            _UxGT("Z Wycof. mm")
+#define MSG_CONTROL_RETRACTF                _UxGT("Wycofaj  V")
+#define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Skok Z mm")
+#define MSG_CONTROL_RETRACT_RECOVER         _UxGT("Cof. wycof. +mm")
+#define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("Z Cof. wyc. +mm")
+#define MSG_CONTROL_RETRACT_RECOVERF        _UxGT("Cof. wycof.  V")
+#define MSG_AUTORETRACT                     _UxGT("Auto. wycofanie")
+#define MSG_FILAMENTCHANGE                  _UxGT("Zmien filament")
+#define MSG_INIT_SDCARD                     _UxGT("Inicjal. karty SD")
+#define MSG_CNG_SDCARD                      _UxGT("Zmiana karty SD")
+#define MSG_ZPROBE_OUT                      _UxGT("Sonda Z za lozem")
+#define MSG_HOME                            _UxGT("Home")  // Used as MSG_HOME " " MSG_X MSG_Y MSG_Z " " MSG_FIRST
+#define MSG_FIRST                           _UxGT("first")
+#define MSG_ZPROBE_ZOFFSET                  _UxGT("Offset Z")
+#define MSG_BABYSTEP_X                      _UxGT("Babystep X")
+#define MSG_BABYSTEP_Y                      _UxGT("Babystep Y")
+#define MSG_BABYSTEP_Z                      _UxGT("Babystep Z")
+#define MSG_ENDSTOP_ABORT                   _UxGT("Blad krancowki")
+#define MSG_HEATING_FAILED_LCD              _UxGT("Rozgrz. nieudane")
+#define MSG_ERR_REDUNDANT_TEMP              _UxGT("Błąd temperatury")
+#define MSG_THERMAL_RUNAWAY                 _UxGT("Zanik temp.")
+#define MSG_ERR_MAXTEMP                     _UxGT("Err: max. temp.")
+#define MSG_ERR_MINTEMP                     _UxGT("Err: min. temp.")
+#define MSG_ERR_MAXTEMP_BED                 _UxGT("Err: max. temp. loza")
+#define MSG_ERR_MINTEMP_BED                 _UxGT("Err: min. temp. loza")
+#define MSG_HEATING                         _UxGT("Rozgrzewanie...")
+#define MSG_HEATING_COMPLETE                _UxGT("Rozgrzano")
+#define MSG_BED_HEATING                     _UxGT("Rozgrzewanie loza...")
+#define MSG_BED_DONE                        _UxGT("Rozgrzano loze")
+#define MSG_DELTA_CALIBRATE                 _UxGT("Kalibrowanie Delty")
+#define MSG_DELTA_CALIBRATE_X               _UxGT("Kalibruj X")
+#define MSG_DELTA_CALIBRATE_Y               _UxGT("Kalibruj Y")
+#define MSG_DELTA_CALIBRATE_Z               _UxGT("Kalibruj Z")
+#define MSG_DELTA_CALIBRATE_CENTER          _UxGT("Kalibruj środek")
```

Comment:
Is `_UxGT()` really needed for all languages, even those that just use Roman and accented characters? I was under the impression these would only need to be applied where the unreadable character codes were being used.